R/datadir2fnames.R

```diff
@@ -12,8 +12,24 @@ datadir2fnames = function(datadir,filelist) {
       fnamesfull = fnamesRD
     }
   } else {
-    fnamesfull = datadir
-    fnames = basename(fnamesfull)
+    if (ismovisens(datadir)) {
+      fnamesfull = dir(datadir, recursive = TRUE, pattern = "acc.bin", full.names = TRUE)
+      fnames = basename(dirname(fnamesfull))
+      nfolders = length(dir(datadir))
+      if (nfolders > length(fnamesfull)) { # meaning there are participants folder without acc.bin
+        # folders without acc.bin
+        allfolders = dir(datadir, full.names = TRUE)
+        foldersWithAccBin = dirname(fnamesfull)
+        noAccBin = allfolders[which(!allfolders %in% foldersWithAccBin)]
+        warning(paste0("The following participant folders do not contain the ",
```

Comment:
Replace "participant folders" => "movisense data folders" to clarify that this is about movisens data.